index.tsx

```diff
@@ -206,7 +202,10 @@ const ATTRIBUTE_TO_JSX_PROP_MAP = [
   'srcSet',
   'tabIndex',
   'useMap',
-].reduce((obj, x) => ((obj[x.toLowerCase()] = x), obj), { for: 'htmlFor' })
```

Comment:
IntelliJ was complaining about [comma expressions](https://www.jetbrains.com/help/phpstorm/javascript-and-typescript-comma-expression.html)